tests/analysis/AnalysisTests.cmake

```diff
@@ -64,8 +64,19 @@ macro(artifact_handler CATEGORY INPUT_FILE CONFIGURATION OUTPUT TARGET_NAME)
         "${INPUT_FILE}"
         "${OUTPUT}"
         Lift
+        Lift
+        --analysis
```

Comment:
Didn't we say we want two separate invocations?

Reply:
not sure what you mean, it is divided into two steps, there is the && in the middle

we must provide both the step and the analysis name, because otherwise we don't know from which step we want to extract the output. I guess i can make it so that analysis name are aliases for their step, but i don't like much to be honest

Reply:
Ah, I didn't notice, I'm dumb.